x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/pages/coverage_overview/helpers.ts

```diff
@@ -22,3 +32,39 @@ export const getCardBackgroundColor = (value: number) => {
     }
   }
 };
+
+export const formatRuleFilterOptions = <
```

Comment:
nit: “format” is so generic so it’s unclear what the function does. I’d rename it to `extractSelected(options: EuiSelectableOption[]): string[]`.